_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { AccentedCard } from './accented-card';
+import Gridicon from 'components/gridicon';
+
+export function SingleProductBackup( props ) {
+	const { sitePlans } = props;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<>
+			<h1 className="plans-section__header">Single Products</h1>
+			<h2 className="plans-section__subheader">Just looking for backups? We've got you covered.</h2>
+			<div style={ { display: 'flex', justifyContent: 'center', marginBottom: '10px' } }>
+				<AccentedCard>
+					{ {
+						header: <SingleProductBackupHeader sitePlans={ sitePlans } />,
+						body: (
+							<SingleProductBackupBody
+								selectedBackupType={ selectedBackupType }
+								setSelectedBackupType={ setSelectedBackupType }
+								sitePlans={ sitePlans }
+							/>
+						),
+					} }
+				</AccentedCard>
+			</div>
+		</>
+	);
+}
+
+function SingleProductBackupHeader( props ) {
+	const { sitePlans } = props;
+
+	return (
+		<div
+			style={ {
+				display: 'flex',
+				flexDirection: 'row',
+				justifyContent: 'space-between',
+				alignContent: 'center',
+			} }
+		>
+			<h3>{ __( 'Jetpack Backup' ) }</h3>
+			{ sitePlans && (
+				<PlanPriceDisplay
```

Comment:
I'm not sure what you mean by automated. The code is pulling the prices from the dummy data on that `sitePlans` object, which I made to be the same shape as the other responses from that endpoint. So even when the prices are added to the server at some point the code will still have to access `price.yearly.text` and `price.yearly.amount`.

Reply:
What I mean is that we should access a list of the 2 products and loop through that list, instead of manually listing the same component twice with props inputted manually. Does that make sense?

Reply:
Unfortunately I'm not following. Which component are you referring to here?

Reply:
I think I meant to comment this somewhere else. I guess you'll see what I'm talking about when you start using the products endpoint - data is just shaped differently there.